.gitignore

```diff
@@ -10,3 +10,5 @@ target
 .*.swp
 *.vim
 .ensime*
+.bsp
```

Comment:
Intellij creates this build server protocol directory when importing projects with newer versions of SBT.